packages/hash/frontend/src/blocks/page/ComponentView.tsx

```diff
@@ -151,13 +154,13 @@ export class ComponentView implements NodeView<Schema> {
           <BlockLoader
             sourceUrl={this.sourceName}
             blockEntityId={entityId}
-            shouldSandbox={!this.editable}
+            blockMetadata={this.componentMetadata}
+            // shouldSandbox={!this.editable}
```

Comment:
Is commenting this out temporary?

Reply:
Done in https://github.com/hashintel/hash/pull/752/commits/359c3d56034d195e626e9eaa7fe0ffdbc3079480